resource/index.go

```diff
@@ -55,33 +55,31 @@ func (r *index) Compile() error {
 //
 // If a parent is given and the path starts with a dot, the lookup is started at the
 // parent's location instead of root's.
-func (r *index) GetResource(path string, parent *Resource) (*Resource, bool) {
+func (r *index) GetResource(path string, parent *Resource) (sr *Resource, found bool) {
 	resources := r.resources
 	if len(path) > 0 && path[0] == '.' {
 		if parent == nil {
 			// If field starts with a dot and no parent is given, fail the lookup
-			return nil, false
+			return
```

Comment:
Apparently, this is called a "naked return".

Quoting https://tour.golang.org/basics/7:

> Naked return statements should be used only in short functions, as with the example shown here. They can harm readability in longer functions.